src/app/component/AddToCart/AddToCart.component.js

```diff
@@ -52,14 +52,16 @@ export default class AddToCart extends PureComponent {
             product: { type_id },
             isLoading,
             buttonClick,
+            disabledButtonClick,
             isDisabled
         } = this.props;
 
         if (!type_id) this.renderPlaceholder();
 
         return (
+        <div block="ProductActions-AddToCart AddToCart" elem="ButtonWrapper" onClick={isDisabled ? disabledButtonClick : () => {}}>
```

Comment:
Wrapper addition does not seem right to me. You should not over-complicate HTML. Please try using button only.

Reply:
Also, I do recommend not to add `() => {}` things like this in the code. Very misleading. It is preferable to **check for `isDisabled` in the `buttonClick` function**. Also, are you sure `isDisabled` only covers your case? I think the out of stock as well. Make sure the notification is shown only in your case.